doc/classes/EditorSettings.xml

```diff
@@ -408,9 +488,26 @@
 		<member name="editors/animation/onion_layers_past_color" type="Color" setter="" getter="">
 			The modulate color to use for "past" frames displayed in the animation editor's onion skinning feature.
 		</member>
+		<member name="editors/bone_mapper/handle_colors/error" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/bone_mapper/handle_colors/missing" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/bone_mapper/handle_colors/set" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/bone_mapper/handle_colors/unset" type="Color" setter="" getter="">
+		</member>
+		<member name="editors/grid_map/editor_side" type="int" setter="" getter="">
+			Specifies the side of 3D editor's viewport where GridMap's mesh palette will appear.
+		</member>
+		<member name="editors/grid_map/palette_min_width" type="int" setter="" getter="">
+			Minimum width of GridMap's mesh palette side panel.
```

Comment:
In pixels? Relative to the editor's scale?

Reply:
Unscaled pixels. Though its the size of *inside* of the panel, not the whole panel.